atlas/AtlasMl/atlasml/ml/MLPipelines/PipelineWorkflows.py

```
@@ -0,0 +1,292 @@
+                "name": competency["properties"]["name"],
+                "text": competency["properties"]["text"],
+                "cluster_id": clusters[best_medoid_idx]["properties"]["cluster_id"],
+                "cluster_similarity_score": similarity_score[best_medoid_idx]
+            }
+            self.weaviate_client.update_property_by_id(CollectionNames.COMPETENCY.value, competency["id"], properties)
+
+        for index in range(len(texts)):
+            text_entry = texts[index]
+            competency = self.weaviate_client.get_embeddings_by_property(CollectionNames.COMPETENCY.value, "cluster_id",
+                                                                         str(labels[index]))
+            if not competency: continue
```

Ruff (0.11.9)

Multiple statements on one line (colon)

(E701)